math/binarymath/checkisnumberpoweroftwo_test.go

```diff
@@ -0,0 +1,92 @@
+// checkisnumberpoweroftwo_test.go
+// description: Test for Is the number a power of two
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see checkisnumberpoweroftwo.go
+
+package binarymath
+
+import (
+	"math"
+	"testing"
+)
+
+func getTestsForPowerOfTwo() []struct {
```

Comment:
The tests are immutable, so you don't really need to create a factory here. But it doesn't matter too much so even if you don't change it its fine.

Reply:
left it as it is